test/Saritasa.Tools.PropertyChangedGenerator.Demo/Models/Employee.cs

```diff
@@ -1,19 +1,27 @@
-﻿namespace Saritasa.Tools.PropertyChangedGenerator.Demo.Models;
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Saritasa.Tools.PropertyChangedGenerator.Demo.Models;
 
 /// <summary>
 /// Employee model.
 /// </summary>
 internal partial class Employee : EditableModel
 {
-    [AlsoNotify(nameof(FullName))]
+    [AlsoNotify(nameof(FullName), nameof(Surname))]
+    [Property<RequiredAttribute>]
+    [Property<MinLengthAttribute>(3)]
     private string firstname;
 
     [AlsoNotify(nameof(FullName))]
+    [Property<RequiredAttribute>]
     private string surname;
 
     [Accessibility(Setter.Internal)]
+    [Property<RequiredAttribute>]
+    [Property<RangeAttribute>(0, 100)]
```

Comment:
Attributes have 1) positional arguments and 2) properties. 
It is understandable how to pass positional arguments. But how do we pass properties?

Maybe we could utilize the use of attribute targets instead of `[PropertyAttribute`1]` wrapper?
Example:
```cs
    [Accessibility(Setter.Internal)]
    [property: Required]
    [property: Range(0, 100)]
    private int age;
```

Reply:
@ko-vasilev @krasninja What do you think about this?

As i see it's possible to implement
`
[AttributeUsage(AttributeTargets.Property, AllowMultiple = false)]
public class MyAttribute : Attribute
{
    public MyAttribute(object obj, bool required, bool positional)
    {
    }
}

public class MyClass
{
    [property: My(5, positional: true)]
    private int age;
}
`

Reply:
I've seen the same approach today. It is also used by Mvvm Toolkit
https://learn.microsoft.com/en-us/dotnet/communitytoolkit/mvvm/generators/observableproperty#adding-custom-attributes

Reply:
Reference code:
https://github.com/CommunityToolkit/dotnet/blob/main/src/CommunityToolkit.Mvvm.CodeFixers/FieldReferenceForObservablePropertyFieldCodeFixer.cs

Reply:
@danyducky 
Reference code:
https://github.com/CommunityToolkit/dotnet/blob/7b53ae23dfc6a7fb12d0fc058b89b6e948f48448/src/CommunityToolkit.Mvvm.SourceGenerators/ComponentModel/ObservablePropertyGenerator.Execute.cs#L211-L217